### DISCUSSION TITLE
Map Sources and Styles

### DISCUSSION THREAD

**Discussion by psplawski** [OvertureMaps/data#334] [Discussion]
Hey, I am currently in a process of moving maps to Overture Maps and I got following questions:

**Sources:** 
Instead of creating `roads.pmtiles` and `placenames.pmtiles` sources, I am looking for roads and places directly in `transportation.pmtiles` and `places.pmtiles`.

I wanted to ask if this may produce some problems in the future? Right now I was forced by this decision to change source-layer value from roads to segment in case of all items from `roads.pmtiles` in order to make them visible. 

I am struggling with places tho, I am not able to make any of the labels visible on the map, I got a feeling that I am targeting wrong source-layer in my style.json.

**Styles:**
I am curious if there is any style.json example of styles which will look similar to → https://github.com/mapbox/osm-bright or https://docs.protomaps.com/basemaps/themes#light ? 
If yes, I will be more than happy if someone is able to provide me a decent looking style.json that will work with Overture sources.

**Comment by danabauer** [comment]
Hi @psplawski. I'm thrilled to tell you that there are now [Protomaps basemaps](https://github.com/protomaps/basemaps) created from Overture data thanks to @migurski!
